package.json

```diff
@@ -102,15 +102,15 @@
     "@shopify/postcss-plugin": "^5.0.1",
     "@shopify/prettier-config": "^1.1.2",
     "@shopify/react-testing": "^3.2.4",
-    "@shopify/storybook-a11y-test": "^0.0.1",
+    "@shopify/storybook-a11y-test": "^0.3.0",
```

Comment:
I think the way accessibility testing is used has changed from [v0.0.1](https://www.npmjs.com/package/@shopify/storybook-a11y-test/v/0.0.1) to [v0.3.0](https://www.npmjs.com/package/@shopify/storybook-a11y-test/v/0.3.0). Updating per the below will probably get the CI check to pass.

In `scripts/accessibility-check.js`, we should import `{testPages, getCurrentStoryIds}` instead of `storybookA11yTest` on [line 4](https://github.com/Shopify/polaris-react/blob/e25973358f4bcb9c65ef6147cd105de0d4044d9e/scripts/accessibility-check.js#L4).

Then use `await getCurrentStoryIds` to get `storyIds` (previously named variable [options](https://github.com/Shopify/polaris-react/blob/e25973358f4bcb9c65ef6147cd105de0d4044d9e/scripts/accessibility-check.js#L7)). And `await testPages` with those `storyIds` to get [results](https://github.com/Shopify/polaris-react/blob/e25973358f4bcb9c65ef6147cd105de0d4044d9e/scripts/accessibility-check.js#L16).